Add more info on why not use 1337 for app pod (#9462)
istio/istio.io: content/en/docs/ops/deployment/requirements/index.md

  instance HTTP and TCP.

- **Application UIDs**: Ensure your pods do **not** run applications as a user

>   with the user ID (UID) value of **1337**. (`1337` is reserved for the sidecar proxy).

Reviewer: ```suggestion
  with the user ID (UID) value of `1337` because `1337` is reserved for the sidecar proxy.
```

>   with the user ID (UID) value of **1337**. The `istio-proxy` container runs as UID **1337**. Running your application containers using the same UID would result in conflicts with its configuration.

Reviewer 2: ```suggestion
  with the user ID (UID) value of **1337**. (1337 is reserved for the sidecar proxy).
```

Author: Thanks

>   with the user ID (UID) value of **1337**. The proxy container runs as UID 1337, there iptables rules are configured so that all traffic coming from processes with UID 1337 bypass the proxy. If an application runs under 1337, it also bypasses the proxy that's why you shouldn't use 1337 for your application container.

Reviewer 3: ```suggestion
  with the user ID (UID) value of **1337**. The istio-proxy container runs as UID 1337. Running your application containers using the same UID would result in conflicts with its iptables configuration. 
```
